novawallet/Common/URLHandling/UrlHandlingAction.swift

```diff
@@ -1,6 +1,25 @@
 import Foundation
 
-enum UrlHandlingAction: String {
-    case open
-    case create
+enum UrlHandlingAction {
+    case open(screen: String)
+    case create(screen: String)
+
+    init?(from url: URL) {
+        let pathComponents = url.pathComponents
+        let action = pathComponents[1].lowercased().trimmingCharacters(in: .whitespacesAndNewlines)
```

Comment:
I think we should check whether path components at a given index exists. Otherwise we will crash.